generate/unix/Makefile.config

```diff
@@ -227,22 +227,39 @@ CWARNINGFLAGS += \
     -Wold-style-definition\
     -Wno-format-nonliteral\
     -Wredundant-decls
+
+# output directory for tests below
 #
-# Per-host flags and exclusions
+# Taken from the Linux kernel
+TMPOUT = $(if $(KBUILD_EXTMOD),$(firstword $(KBUILD_EXTMOD))/).tmp_$$$$
```

Comment:
KBUILD_EXTMOD does not make any sense outside of the Linux kernel's build system